packages/common/src/discussions/_internal/computeProps.ts

```diff
@@ -25,6 +26,8 @@ export function computeProps(
     const session: UserSession = requestOptions.authentication as UserSession;
     token = session.token;
   }
+  // compute base properties on discussion
+  discussion = computeBaseProps(model.item, discussion);
```

Comment:
This is great! The one thing I've noticed is that discussions don't have an `enrichDiscussionSearchResult` function, meaning they are relying on the base `enrichContentSearchResult` logic. We should probably let engagement know and verify that this was intentional and not accidental.